flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/NettyShuffleEnvironmentConfigurationTest.java

```diff
@@ -72,145 +85,6 @@ public void testNetworkRequestBackoffAndBuffers() {
 		assertEquals(networkConfig.floatingNetworkBuffersPerGate(), 100);
 	}
 
-	/**
-	 * Test for {@link NettyShuffleEnvironmentConfiguration#calculateNetworkBufferMemory(long, Configuration)} using old
```

Comment:
`NettyShuffleEnvironmentConfiguration#calculateNetworkBufferMemory` is till not removed
could be part of
b5d4267e [FLINK-13986][core][runtime] Remove java codes for legacy memory calculations.